schemas/applications/schema/events/load.schema.json

```diff
@@ -1,18 +1,51 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "title": "Load Component",
-  "description": "Load a component",
+  "title": "Load",
+  "description": "Load a component, hardware interface or controller",
   "oneOf": [
     {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
       "items": {
-        "$ref": "identifiers/component.schema.json"
+        "$ref": "#/$defs/load_object"
       }
     },
     {
-      "$ref": "identifiers/component.schema.json"
+      "$ref": "#/$defs/load_object"
     }
-  ]
+  ],
+  "$defs": {
+    "load_object": {
+      "oneOf": [
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "component": {
+              "$ref": "identifiers/component.schema.json"
+            }
+          },
+          "required": [
+            "component"
+          ]
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "hardware": {
+              "$ref": "identifiers/hardware.schema.json"
+            }
+          },
+          "required": [
+            "hardware"
+          ]
+        },
+        {
+          "$ref": "identifiers/controller.schema.json"
+        }
```

Comment:
I was trying to think of a more compact way to define these three cases (since it's also repeated in `unload`, but because the component and hardware identifiers are just strings, you do indeed need to define an object around each of them, so I think it's good 👍